themes/demo-theme/variables.json

```diff
@@ -164,7 +163,6 @@
 			"bg": "#fcf8e3"
 		}
 	},
-	"spacer": "1rem",
 	"sizes": {
```

Comment:
Sizes need to be updated. https://devtopia.esri.com/Beijing-R-D-Center/ExperienceBuilder/pull/17853